packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -212,17 +219,29 @@ export class ModelScene extends Scene {
    * Sets the point in model coordinates the model should orbit/pivot around.
    */
   setTarget(modelX: number, modelY: number, modelZ: number) {
-    this.model.position.set(-modelX, -modelY, -modelZ);
-    this.isDirty = true;
+    this.goalTarget.set(-modelX, -modelY, -modelZ);
   }
 
-  /**
-   * The AR scene is anchored at the center of the lower, forward edge of the
-   * bounding box.
-   */
-  setARTarget() {
-    const {min, max} = this.model.boundingBox;
-    this.model.position.set(-(min.x + max.x) / 2, -min.y, -max.z);
+  getTarget(): Vector3 {
+    return vector3.copy(this.goalTarget).multiplyScalar(-1);
+  }
+
+  jumpToGoal() {
+    this.updateTarget(10000);
```

Comment:
![image](https://i.giphy.com/media/tPKoWQJk3cEbC/200.gif)

...seriously though: let's document this magic number by assigning it to a module-level constant!